includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,710 @@
+				],
+				'default' => '#ff0000',
+			]
+		);$this->add_responsive_control(
+			'club_color',
+			[
+				'label' => esc_html__( 'Club ', 'elementor' ) . static::$suit_char_map['club'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-club-color: {{VALUE}}',
+				],
+				'default' => '#000000',
+			]
+		);$this->add_responsive_control(
+			'spade_color',
+			[
+				'label' => esc_html__( 'Spade ', 'elementor' ) . static::$suit_char_map['spade'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-spade-color: {{VALUE}}',
+				],
+				'default' => '#000000',
+			]
+		);
+		$this->end_controls_section();
+		$this->start_controls_section(
+			'royal_suit_offset',
+			[
+				'label' => esc_html__( 'Royal Suit Offset', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_STYLE,
+			]
+		);
+		$this->add_responsive_control(
+			'royal_offset_h',
+			[
+				'label' => esc_html__( 'Horizontal Offset', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'em', 'rem', '%', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 50,
+						'step' => 1,
+					],
+					'em' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'rem' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'%' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-royal-suit-offset-h: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+		$this->add_responsive_control(
+			'royal_offset_v',
+			[
+				'label' => esc_html__( 'Vertical Offset', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'em', 'rem', '%', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 50,
+						'step' => 1,
+					],
+					'em' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'rem' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'%' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-royal-suit-offset-v: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+		$this->end_controls_section();
+
+	}
+
+	protected function register_controls() {
+		$this->register_cards_section_controls();
+		$this->register_layout_section_controls();
+		$this->register_card_style_controls();
+		$this->register_value_controls();
+		$this->register_suit_controls();
+	}
+
+	private static function render_card( $value, $suit ) {
+		$value = (int) $value;
+		$text = static::$value_map[ $value ];
+		$suit = static::$suit_char_map[ $suit ];
+		$class = __( 'e-card-content', 'elementor' );
+		?>
+		<div class="<?php echo esc_html( 'e-card-backside' ); ?>">
+			<div class="image"></div>
```

Comment:
We prefer to use nestable class names.

```suggestion
			<div class="e-card-image"></div>
```